Sources/NautilusTelemetry/Exporters/Exporter+Trace.swift

```diff
@@ -91,8 +91,21 @@ extension Exporter {
 	}
 
 	func buildLinks(_ span: Span) -> [OTLP.SpanLink]? {
-		guard let linkedParent = span.linkedParent else { return nil }
-		return [OTLP.SpanLink(traceId: linkedParent.traceId, spanId: linkedParent.id)]
+		span.links?.map { link in
+			let attributes: [OTLP.V1KeyValue]?
+			let key = "relationship"
+
+			switch link.relationship {
+			case .undefined:
+				attributes = nil
```

Comment:
Does this mean that an undefined link won't be sent to the backend and is only available to be read locally? I am asking to learn @ladd .